subsys/net/ip/Kconfig.mgmt

```diff
@@ -25,7 +25,7 @@ if NET_MGMT_EVENT
 
```

Comment:
One minor nit-pick, the commit subject should start with "net: " as the subsys: is just unnecessary information here.